src/chat/participant.ts

```diff
@@ -160,7 +162,11 @@ export async function handleChatMessage(
 
   // inform the model that tools can be invoked as part of the response stream
   const requestOptions: LanguageModelChatRequestOptions = {
-    tools: [new ListTemplatesTool().toChatTool()],
+    tools: [
+      new ListTemplatesTool().toChatTool(),
+      new ApplyTemplateTool().toChatTool(),
+      new GetProjectTemplateTool().toChatTool(),
```

Comment:
this `GetProjectTemplateTool` returns targeted info from the templates list so that the `ApplyTemplateTool` can fire